docs/install.md

```diff
@@ -137,5 +137,6 @@ Installing using custom recoveries is only possible if your device has boot ramd
 - Rename the `.apk` file extension to `.zip`, for example: `Magisk-v24.0.apk` → `Magisk-v24.0.zip`. If you have trouble renaming the file extension (like on Windows), use a file manager on Android or the one included in the custom recovery to rename the file.
 - Flash the zip just like any other ordinary flashable zip.
 - Reboot and check whether the Magisk app is installed. If it isn't installed automatically, manually install the APK.
+- Launch the Magisk app, and it will show a dialog asking for a reinstallation. Do the reinstallation **within app** and reboot (if you are using MTK devices that lock the boot partition after boot, please [patch the boot image](#patching-images) and flash it by custom recovery or fastboot).
```

Comment:
```suggestion
- Launch the Magisk app, and it will show a dialog asking for a reinstallation. Do the reinstallation **within app** and reboot (if you are using MTK devices that lock the boot partition after boot, please [patch the boot image](#patching-images) and flash it by custom recovery or fastboot).
- For advanced users, you can set `PREINITDEVICE` environment variable before flash.
```